python/pyxbos/pyxbos/drivers/hvac/point_values_by_equipment.json

```diff
@@ -0,0 +1,429 @@
+{
```

Comment:
Is this file used in the driver? If not, we should remove it

Reply:
No, but I moved all the json files into a subdirectory for orinda-public-library